units/provisioning_create_host.go

```diff
@@ -122,23 +122,32 @@ func (j *createHostJob) createHost(ctx context.Context) error {
 		return errors.Wrapf(errIgnorableCreateHost, "problem getting cloud provider for host '%s' [%s]", j.host.Id, err.Error())
 	}
 
-	// On the first attempt, remove the intent document so no other create host job tries to create this intent.
+	defer j.tryRequeue(ctx)
+
+	if err := j.host.SetStatus(evergreen.HostBuilding, evergreen.User, ""); err != nil {
```

Comment:
Could you add a comment here explaining how HostBuilding works in the context of the job? I don't understand the change.